src/NewProject/source_grid.cpp

```diff
@@ -14,12 +14,12 @@ using namespace FOEDAG;
 
 sourceGrid::sourceGrid(QWidget *parent) : QWidget(parent) {
   m_lisFileData.clear();
-  m_btnAddFile = new QPushButton(tr("AddFile"), this);
+  m_btnAddFile = new QPushButton(tr("AddFile..."), this);
   connect(m_btnAddFile, &QPushButton::clicked, this, &sourceGrid::AddFiles);
-  m_btnAddDri = new QPushButton(tr("AddDir"), this);
+  m_btnAddDri = new QPushButton(tr("AddDir..."), this);
   connect(m_btnAddDri, &QPushButton::clicked, this,
           &sourceGrid::AddDirectories);
-  m_btnCreateFile = new QPushButton(tr("CreateFile"), this);
+  m_btnCreateFile = new QPushButton(tr("CreateFile..."), this);
```

Comment:
Can you add a screenshot for this change ?

Reply:
![rew](https://user-images.githubusercontent.com/109368108/180590832-59692d72-5057-4cda-a9d5-17225614f67a.png)

Reply:
> @zaryabkhan-rs the screenshot is outdated after the previous PR "Move Up" and "Move Down" change from previous PR (#542). "AddFile..." -> "Add File..." "AddDir..." -> "Add Directory..." "CreateFile..." -> "Create File..." FYI: @skyler-rs

As It was a separate issue and checkout from main, that's why those changes aren't merged with this PR. It should merge with #542 once this PR is merged.

Reply:
> @ravic-rs ![rew](https://user-images.githubusercontent.com/109368108/180590832-59692d72-5057-4cda-a9d5-17225614f67a.png)

Updated Screenshot!
![dir](https://user-images.githubusercontent.com/109368108/180825625-d17ae6a5-1ff2-4f42-90aa-05134416a639.png)

Reply:
@ravic-rs  Updated Screenshot ! 
![upd](https://user-images.githubusercontent.com/109368108/180828166-427e9937-277c-48c4-88f0-ce4c41cd750b.png)

Reply:
Thanks!